react/features/base/dialog/components/native/ConfirmDialog.js

```diff
@@ -49,6 +49,11 @@ type Props = {
      * Dialog title.
      */
     title?: string,
+
+    /**
+     * Dialog visibility.
+     */
+    visible?: Boolean
```

Comment:
Please drop this, dialog visibility is already controlled with openDialog hideDialog